src/Form/RenderFileTab.tsx

```diff
@@ -84,14 +111,27 @@ export function RenderBasicFileTab({ step, splitSchema, setSplitSchema }: Render
 
   return (
     <Box sx={{ pb: 4, pt: 4 }}>
-      <Stack direction='row' spacing={2} alignItems='center'>
-        <FileInput label='Select Code' file={code} onChange={handleCodeChange} accepts='.zip' />
-        <FileInput label='Select Binary' file={binary} onChange={handleBinaryChange} accepts='.zip' />
-      </Stack>
+      {(buildOptionsStep.state.uploadType === undefined ||
```

Comment:
See https://github.com/gchq/Bailo/pull/139#discussion_r975295397 - same concept could be applied here